backend/server/Cargo.toml

```diff
@@ -18,20 +18,24 @@ embed-in-debug = ["reinda/debug-is-prod"]
 
 [dependencies]
 anyhow = "1"
+base64 = "0.13"
 bunt = "0.2.4"
 chrono = "0.4"
 deadpool-postgres = { version = "0.7", default-features = false }
 futures = "0.3.1"
 hostname = "0.3"
 hyper = "0.13"
+hyper14 = { package = "hyper", version = "0.14", features = ["client", "http1", "http2"] }
```

Comment:
Because juniper still has not updated to tokio 1.0 and hyper 0.14. It's a mess, I know. Every time I see it, I check the status of juniper. But yeah, its maintenance is in a rough shape. But this is nothing I can fix in this PR. And using a deliberately older version of hyper in this code just to match what another dependency requires is also meh. We will update sooner or later.